src/components/NewCommentForm.tsx

```diff
@@ -75,25 +160,37 @@ export const NewCommentForm: React.FC = () => {
             id="comment-body"
             name="body"
             placeholder="Type comment here"
-            className="textarea is-danger"
+            className={classNames('textarea', { 'is-danger': bodyError })}
+            value={body}
+            onChange={(event) => setBody(event.target.value)}
           />
         </div>
 
-        <p className="help is-danger" data-cy="ErrorMessage">
-          Enter some text
-        </p>
+        {bodyError && (
```

Comment:
```suggestion
        {!!bodyError && (
```
in rendering with &&  need to convert the value to boolean.